src/components/AccountIndex.tsx

```diff
@@ -0,0 +1,21 @@
+import type { AccountIndex as PAccountIndex } from '@polkadot/types/interfaces'
+import { useEffect, useState } from 'react'
+import { useKusama } from '../kusama'
+
+const AccountIndex = ({ member } : { member: SocietyMember }) => {
+  const { api } = useKusama()
+  const [index, setIndex] = useState<string>('Not set')
```

Comment:
What about an empty string here? It would make the UI cleaner in case we have many not-set values.

Reply:
Agreed!